### ISSUE TITLE
Why did `object-alt` and `aria-required-children` stopped failing in 4.5.0?

### ISSUE THREAD

**Issue by adamraine** [dequelabs/axe-core#3887]
### Product

axe-core

### Question

In axe-core 4.4.3 the following page would fail on `object-alt` and `aria-required-children`:

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Test a11y</title>
</head>
<body>
  <h1>Test a11y</h1>
  <object id="object-alt"></object> 
  <div id="aria-required-children" role="radiogroup">
    <div></div>
  </div>
</body>
</html
```

However, in version >=4.5.0 no failures are reported on this page. I couldn't find anything the [changelog](https://github.com/dequelabs/axe-core/releases/tag/v4.5.0) to explain why this happened, but I'm assuming I missed something. What changes in version 4.5.0 caused this to happen?

**Comment by straker** [comment]
Thanks for the question. For object-alt, we added a fix to ignore [objects without a `data` attribute](https://github.com/dequelabs/axe-core/pull/3680), and or aria-required-children we added a fix to [remove the `requiredOwned` for radiogroup](https://github.com/dequelabs/axe-core/pull/3597).

**Comment by adamraine** [comment]
Thanks for the quick response @straker!
